tests/resources/test_files_resource.py

```diff
@@ -385,3 +392,40 @@ def test_disabled_upload_file_resource(client, es_clear, headers, location):
         f"/mocks_disabled_files_upload/{id_}/files/test.pdf", headers=headers
     )
     assert res.status_code == 405
+
+
+def test_disable_files_when_files_already_present_should_error(
+        app, client, es_clear, headers, input_data, location):
+    # Initialize a record
+    response = client.post('/mocks', headers=headers, json=input_data)
```

Comment:
Question: is it always required to send ``input_data`` now?

Reply:
For the `RecordWithFilesSchema`, the `files` attribute is required so `input_data` is required (with `files` filled). It could be made optional... It introduced some questions I wasn't sure how to answer: default to True probably? but at the component level or at the schema level? and where can we assume that we have a value for that field - don't want to always be checking.... Forcing `required` was explicit :sweat_smile: . Not attached to any approach though.